web/packages/teleterm/src/deepLinks.test.ts

```diff
@@ -213,26 +213,6 @@ describe('parseDeepLink followed by makeDeepLinkWithSafeInput gives the same res
     'teleport://cluster.example.com/connect_my_computer',
     'teleport://alice@cluster.example.com/connect_my_computer',
     'teleport://alice.bobson%40example.com@cluster.example.com:1337/connect_my_computer',
```

Comment:
this is just combining the two tests and making sure to check url.searchParams (i originally duplicated the test)